docs/requirements-docs.txt

```diff
@@ -8,6 +8,11 @@ sphinx_autosummary_accessors
 sphinx-tabs
 sphinx-design
 jupyter_sphinx
+sphinxcontrib-applehelp>=1.0.0,<1.0.7
```

Comment:
A comment here with a little more context would be good (mostly so we know when we can remove these pins)

```suggestion
# FIXME: `sphinxcontrib-*` pins are a workaround until we have sphinx>=5.
#        See https://github.com/dask/dask-sphinx-theme/issues/68.
sphinxcontrib-applehelp>=1.0.0,<1.0.7
```

Reply:
I went ahead and committed this minor change (hope that's okay @milesgranger)